README.md

```diff
@@ -1,12 +1,10 @@
 # Accessible autocomplete
 
----
-
-**We cannot currently respond to issues or pull requests.**
+The accessible autocomplete is a component that helps users choose countries and territories in answers.
```

Comment:
@edwardhorsford Would this be a better description for the country-and-territory autocomplete? Am concerned using the same description in both READMEs risks confusing users.

Reply:
lol no - this is explicitly not the component to use for picking countries and territories 😆